src/test/java/LoinPageTest.java

```diff
@@ -0,0 +1,81 @@
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pages.ForgotYourPasswordPage;
+import pages.LoginPage;
+
+import java.time.Duration;
+
+public class LoinPageTest {
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/drivers/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+        driver.get("https://magento.softwaretestingboard.com/customer/account/login/");
+    }
+
+    @Test
+    public void loginPage_LeftPanel_HasTexDisplayedAndSignInButtonEnabled() {
+        //given
+        LoginPage loginPage = new LoginPage(driver);
+
+        //then
+        Assert.assertTrue(loginPage.isTextPageTitleDisplayed());
```

Comment:
Please do not mix with Assert from testng and Assertions from assertj. Please stick to only one - in this case assertj. It should be `Assertions.assertThat(loginPage.isTextPageTitleDisplayed()).isTrue();`